### ISSUE TITLE
Stretch Goal: Adding audio to front site

### ISSUE THREAD

**Issue by LukasEberle** [LukasEberle/PySB#3]
Has to be implemented after #2 

I can add audio to a front site entry as `[sound:myaudio.mp3]`, so I want to use the suprisingly decent Japanese Text-to-Speech of Google Translate to add sound cues to the cards. 
I should be able to use the package `gTTS` (https://pypi.org/project/gTTS/). If the package does not support Hiragana, Katakana or Kanji, I can work aroud that with the third column, which contains the romanji spelling, to generate the sound.
